src/main/webapp/app/exam/manage/student-exams/student-exam-detail.component.html

```diff
@@ -147,8 +153,16 @@ <h5><span jhiTranslate="artemisApp.studentExamDetail.exercises">Exercises</span>
                             {{ exercise?.title }}
                         </td>
                         <td class="align-middle">
-                            <ng-container *ngIf="exercise?.studentParticipations && exercise?.studentParticipations.length > 0 && (exercise?.studentParticipations)![0].results">
-                                {{ rounding(((exercise?.studentParticipations)![0].results![0].score * exercise?.maxPoints) / 100) }}
+                            <ng-container
+                                *ngIf="
+                                    exercise?.studentParticipations &&
+                                    exercise?.studentParticipations.length > 0 &&
+                                    exercise?.studentParticipations?.length > 0 &&
+                                    (exercise?.studentParticipations)![0].submissions?.length > 0 &&
+                                    (exercise?.studentParticipations)![0].submissions[0].results
+                                "
```

Comment:
usually this would be enough, but the linter gives an error: `if exercise?.studentParticipations?.[0]?.submissions?.[0]?.results`

```suggestion
                                *ngIf="
                                    exercise?.studentParticipations?.length > 0 &&
                                    exercise.studentParticipations![0].submissions?.length > 0 &&
                                    exercise.studentParticipations![0].submissions[0].results
                                "
```